src/basics.md

```diff
@@ -246,22 +247,25 @@ fn main() {
 
 [![rand-badge]][rand] [![cat-science-badge]][cat-science]
 
-Randomly generates a tuple `(i32, bool, f64)` and variable of user defined type `Point`.
-Implements the [`rand::Rand`] trait for `Point` in order to allow random generation.
+Randomly generates a tuple `(i32, bool, f64)` and variable of user defined type
+`Point`. Implements `Point` for [`rand::distributions::Uniform`] in order to
+allow random generation.
 
 ```rust
 extern crate rand;
 
-use rand::{Rng, Rand};
+use rand::Rng;
+use rand::distributions::{Distribution, Uniform};
 
 #[derive(Debug)]
 struct Point {
     x: i32,
     y: i32,
 }
 
-impl Rand for Point {
-    fn rand<R: Rng>(rng: &mut R) -> Point {
+impl Distribution<Point> for Uniform {
```

Comment:
This line error-ed with expected 1 type argument.  I was able to get it to compile with changing it to `Standard`.